### PR TITLE
Add dependent timeline page AB#14548

### PR DESCRIPTION
**Pull request by MikeLyttle**
# Implements [AB#14548](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14548)

## Description

- adds configuration setting for dependent timeline [AB#14957](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14957)
- adds store for dependents [AB#15011](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/15011)
- updates enabled dependent datasets in local environment
- adds dependent timeline view [AB#14954](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14954)
   - fixes comments appearing on dependent timeline [AB#15010](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/15010)
   - redirects to unauthorized page if dependent is not recognized

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## UI Changes

/dependents/162346565465464564565463257/timeline

localhost-2023-02-24-174619
localhost-2023-02-24-174558

/dependents/badhdid/timeline

localhost-2023-02-24-174533

## Notes

Functional tests still remain ([AB#14747](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14747)) and can be merged to dev seperately.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

File: Apps/WebClient/src/featuretoggleconfig.json [hunk 1]
```diff
@@ -61,11 +61,8 @@
     },
     "dependents": {
         "enabled": true,
+        "timelineEnabled": true,
```
Comment (ytqsl) [inline Apps/WebClient/src/featuretoggleconfig.json:64]: What do you think of changing this to version: 1|2 or version: "card"|"timeline"?

File: Apps/WebClient/src/featuretoggleconfig.json [hunk 2]
```diff
@@ -87,7 +80,7 @@
                 "enabled": false
             },
             {
-                "name": "specialAuthority",
+                "name": "specialAuthorityRequest",
```
Comment (ytqsl) [inline Apps/WebClient/src/featuretoggleconfig.json:83]: I don't remember if this change also needs to happen in the functional tests or not, and in the related EntryType

**Review by ytqsl — APPROVED** [review]
Minor comments, we can discuss later

File: Apps/WebClient/src/featuretoggleconfig.json (hunk 1 above)
Comment (MikeLyttle) [reply Apps/WebClient/src/featuretoggleconfig.json:64]: My understanding is that this setting will no longer be needed after the new functionality goes live.  At that time, the original dependent page can be removed, along with this setting, and the code will go back to checking just the dependents.enabled setting.

**Review by BrianMaki — APPROVED** [review]

**Pull request merged**